### DISCUSSION TITLE
Remove all environment variables at once

### DISCUSSION THREAD

**Discussion by ricardo-fnd** [vercel/community#746] [Help] [answered]
Hey guys, is it possible to delete all environment variables at once in a project via CLI or Vercel interface?
THanks!

**Comment by amyegan** [comment, marked as answer]
Hi @ricardo-fnd. Thanks for reaching out! The web UI doesn't have a way to bulk delete environment variables, but you can set up a script to delete all of your env vars using the CLI. You could use `vercel env ls` to get all of your environment variables, and then loop through them to delete with `vercel env rm [name] [environment]`.

You can also use Vercel API for this by getting project info from [/v9/projects/{idOrName}](https://vercel.com/docs/rest-api#endpoints/projects/find-a-project-by-id-or-name) and then looping through the `env` array and deleting with [/v9/projects/{idOrName}/env/{keyOrId}](https://vercel.com/docs/rest-api#endpoints/projects/remove-an-environment-variable)
